app/services/process_notify_callback.rb

```diff
@@ -7,7 +7,14 @@ def call(notify_reference:, status:)
       return :not_updated unless same_environment? && reference_request_email? && permanent_failure_status?
       return :not_found unless ApplicationReference.exists?(reference_id)
 
-      ApplicationReference.find_by(id: reference_id).update!(feedback_status: 'email_bounced')
+      reference = ApplicationReference.find_by(id: reference_id)
+      reference.update!(feedback_status: 'email_bounced')
+
+      SendNewRefereeRequestEmail.call(
+        application_form: reference.application_form,
+        reference: reference,
+        reason: :email_bounced,
+      )
```

Comment:
Should we wrap this in a transaction with the update?

Reply:
Didn't know this was a thing. Makes sense! Done in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1157/commits/b99f0893026bbf26f8fa4aa4d71778f8492a4756.